CONTRIBUTORS

```diff
@@ -24,3 +24,4 @@ Yuzhe Han
 Thomas Garnier
 Utkarsh Anand
 Tobias Klauser
+Atul Prakash
```

Comment:
I think you need to add yourself under the Google section after Eric Biggers.
Or, if it's not covered by Google copyright, then you also need to add yourself to AUTHORS as individual copyright holder.